src/compiler/checker.ts

```diff
@@ -37124,7 +37124,7 @@ namespace ts {
                     error(node, Diagnostics.Re_exporting_a_type_when_the_isolatedModules_flag_is_provided_requires_using_export_type);
                 }
 
-                if (isImportSpecifier(node) && every(target.declarations, d => !!(getCombinedNodeFlags(d) & NodeFlags.Deprecated))) {

```

Comment:
Our `every` helper doesn't handle `undefined`? Most of our other array helpers treat `undefined` as an empty array input. Maybe we should change the helper?

Reply:
It does handle `undefined`, but `every(undefined, _) ==> true`, and then `addDeprecatedSuggestion` crashes on the next line. 

This fix is shorter and uses the standard utility, so I think it's better.

Reply:
Pffffff okay. Yeah.